docker/devbox.dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.11-buster
+FROM python:3.12-buster
```

Comment:
It looks like this image doesn't exist because "buster" is no longer supported. Can you update this to be [bookworm](https://hub.docker.com/layers/library/python/3.12-bookworm/images/sha256-280f63c69e2afdc7344d98269522eccd8c6168b8799e4d855dd07d820d4451d2?context=explore)?